src/app/service/poll-item.service.ts

```diff
@@ -36,14 +36,14 @@ export class PollItemService {
    */
   create<T extends MultipleChoiceItemCreate | QuizItemCreate | OpenTextItemCreate>(pollItem: T): Observable<T> {
```

Comment:
why are there spaces between `|` here then?

Reply:
Reformatted all files. Should be fine now ...